test/models/timdex_test.rb

```diff
@@ -3,7 +3,9 @@
 class TimdexTest < ActiveSupport::TestCase
   def basic_search
     {
-      'q' => 'data'
+      'q' => 'data',
+      'full' => false,
```

Comment:
Full is a REST API feature not GraphQL and shouldn't be necessary here. And until pagination is actually enabled I'm not sure it's super helpful to include it here yet but if you have a reason to include it I'm okay with it as I know we'll need it eventually.